pimd/pim_nb_config.c

```diff
@@ -2833,6 +2834,13 @@ int lib_interface_gmp_address_family_static_group_create(
 				 ifp_name);
 			return NB_ERR_VALIDATION;
 		}
+
+		yang_dnode_get_ip(&group_addr, args->dnode, "./group-addr");
+		if (pim_is_group_224_0_0_0_24(group_addr.ip._v4_addr)) {
+			snprintf(args->errmsg, args->errmsg_len,
+				 "Cannot join multicast range 224.0.0.0/24");
```

Comment:
I know it is obvious for most users, but this error msg might trick some that there is a problem preventing joining the group. Can we make it something like:

```Group 224.0.0.0/24 is reserved and cannot be joined```

Do we also want to prevent the user form configuring subnets within `224.0.0.0/24` ? ex: `224.0.0.0/26`, etc ?

Reply:
Yeah this sounds better. Have changed it.